.github/ISSUE_TEMPLATE/new-integration-proposal.md

```diff
@@ -25,7 +25,7 @@ If the request is accepted, ensure the following checklist is complete before cl
 - [ ] The code is documented with docstrings and was merged in the `main` branch
 - [ ] Docs are published at https://docs.haystack.deepset.ai/
 - [ ] There is a Github workflow running the tests for the integration nightly and at every PR
-- [ ] A label named like `integration:<your integration name>` has been added to this repo
+- [ ] A new label named like `integration:<your integration name>` has been added to the list of labels for this repository
```

Comment:
at this point, we can also add this link for clarity: https://github.com/deepset-ai/haystack-core-integrations/labels